usr/lib/python3/dist-packages/linuxmusterLinuxclient7/printers.py

```diff
@@ -113,12 +117,17 @@ def _getInstalledPrintersOfUser(username):
     return True, installedPrinters
 
 def _uninstallPrinter(name):
-    logging.info("* Uninstalling Printer {}".format(name))
-    uninstallCommand = ["lpadmin", "-x", name]
-    #logging.debug("* running '{}'".format(installCommand))
-
-    if not subprocess.call(uninstallCommand) == 0:
-        logging.error("* Error uninstalling printer!")
-        return False
-
-    return True
+    logging.info("* _uninstallPrinter {}".format(name))
```

Comment:
Is all of this necessary over here? I have never seen a freeze during uninstalling a printer.

Reply:
Bin mir nicht sicher, hatte beim Testen auch den Fall, dass beim Deinstallieren ein Passwort abgefragt wurde.

Reply:
Could you reproduce that again?
I don't think this can happen, since the installation would fail in that case, so the printer would not be uninstalled.

Reply:
Bei einem Server-Cups werden bei uninstall alle dort aufgelisteten Drucker abgearbeitet, auch wenn sie vorher nicht installiert wurden, insofern sicherheitshalber notwendig.

Reply:
Ok, whatever. Let's leave it in, should not hurt anybody.